python-client/giskard/commands/cli_server.py

```diff
@@ -91,6 +92,33 @@ def get_container(version=None, quit_if_not_exists=True) -> Optional[Container]:
             return None
 
 
+def _is_backend_ready(endpoint) -> bool:
+    try:
+        response = requests.get(endpoint)
+        response.raise_for_status()
+        return "UP" == response.json()["status"]
+    except KeyboardInterrupt:
+        raise click.Abort()
+    except BaseException:
+        return False
+
+
+def _wait_backend_ready(port: int) -> bool:
+    endpoint = f"http://localhost:{port}/management/health"
+    backoff_time = 2
+    max_duration_second = 3 * 60
+    started_time = time.time()
+    up = False
+
+    while not up or time.time() - started_time > max_duration_second:
```

Comment:
it should actually be 

```suggestion
    while not up and time.time() - started_time <= max_duration_second:
```